temporian/core/evaluation.py

```diff
@@ -208,7 +208,8 @@ def build_schedule(
     # Operators ready to be computed (i.e. ready to be added to "planned_ops")
     # as all their inputs are already computed by "planned_ops" or specified by
     # "inputs".
-    ready_ops: Set[Operator] = set()
+    ready_ops: List[Operator] = []
+    ready_ops_set: Set[Operator] = set()
```

Comment:
I think it's safe to remove `ready_ops_set` now we got `ready_ops`. I don't see its purpose.

Reply:
"ready_ops" and "ready_ops_set" contain the same data. Those two containers have different properties and costs (one is a list, the other is a set). The list is great for a FILO, while the set is great to check for presence of an item.

>> I think it's safe to remove ready_ops_set now we got ready_ops

You mean there is a situation where this code will not work? If so, can you give details?